model/src/main/proto/profile.proto

```diff
@@ -44,6 +43,17 @@ message Profile {
   int64 date_created_timestamp_ms = 9;
 }
 
+// Structure for profile avatar image
+message ProfileAvatar {
+  oneof avatar_type {
+    // User selected image or gravatar.
+    string avatar_image_uri = 5;
+
+    // Random color used if user does not provide image
```

Comment:
Perhaps instead: 'Tint color to be used with a generic profile image.'

The 'does not provide image' is implied since this is a oneof with the avatar image URI.